process/block/interceptors/metachainHeaderInterceptor.go

```diff
@@ -110,13 +112,13 @@ func (mhi *MetachainHeaderInterceptor) ProcessReceivedMessage(message p2p.Messag
 }
 
 func (mhi *MetachainHeaderInterceptor) processMetaHeader(metaHdrIntercepted *block.InterceptedMetaHeader) {
-	isHeaderOkForProcessing := mhi.headerValidator.IsHeaderValidForProcessing(metaHdrIntercepted.MetaBlock)
-	if !isHeaderOkForProcessing {
-		log.Debug("intercepted meta block header already processed")
+	err := mhi.headerValidator.HeaderValidForProcessing(metaHdrIntercepted)
+	if err != nil {
+		log.Debug(fmt.Sprintf("intercepted meta block header already processed: %s", err.Error()))
```

Comment:
log.Debug("string" + err.Error()) no need for fmt.

Reply:
changed